insights/parsers/tuned_conf.py

```diff
@@ -0,0 +1,92 @@
+"""
+This module contains following parser:
+
+TunedConfIni - file ``/etc/tuned.conf``
+---------------------------------------
+"""
+
+from .. import parser, Parser, get_active_lines
+from insights.specs import Specs
+from insights.contrib.ConfigParser import NoSectionError, NoOptionError
+
+
+@parser(Specs.tuned_conf)
+class TunedConfIni(Parser):
+    """This class parses the ``/etc/tuned.conf`` file using the
+    ``IniConfigFile`` base parser.
```

Comment:
Are you trying to use the `IniConfigFile` parser as its base class? Seems by doing this, this parser can be simplified very much.

Reply:
Current INI parser is not able to handle one corner case "# enabled=False" "<single space>enabled=False", throws error.